examples/finance/bs_ivbp.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<p>While there is indeed an difference between time/t in Devito, I'd stick to use just <code>t</code>  in the math that would look nice and easier to read.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done